src/index.ts

```diff
@@ -118,6 +122,11 @@ class Monika extends Command {
       default: false,
     }),
 
+    prometheus: flags.integer({
+      char: 'p',
```

Comment:
- [ ] I think we can omit this. We might need the `-p` for something in the future. For now, user needs to use `--prometheus` to activate this feature.
- [ ] Also need to mention that this flag requires a number for the port. Probably need to mention the range of the port?
- [ ] Need to add `exclusive:` option as well to specify that this flag cannot be used with the `r` flag.

Reply:
For the `p` flag, description about port, and exclusive flag, I fixed it here [d3a3b78](https://github.com/hyperjumptech/monika/pull/270/commits/d3a3b7819ed4711cb475d4e0357d912fbfdea209). I'm working on the test.

Reply:
I added the test here [883dd88](https://github.com/hyperjumptech/monika/pull/270/commits/883dd88fb8d9c7611f27a44f480a74e934163301).